runtime/rehack/js/runtime.js

```diff
@@ -2,9 +2,8 @@
  * Runtime
  * @providesModule Runtime
  */
-"use strict";
 
-let joo_global_object = global;
```

Comment:
This file is generated by doing `esy test` in the root. There's a file at `rehack_tests/templates/common-js-runtime-header.js` which determines the output of `esy test` which needs to be changed to regenerate this.